cucumber/api/features/step_definitions/logs_steps.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+Given(/^I save the log data from bookmark$/) do
+  lines_amount_in_bookmark()
+end
+
+Given(/^I save the log data from bookmark "([^"]*)"$/) do |bookmark|
```

Comment:
We should be able to give a name to the bookmark in case we want couple of bookmarks also when reading from bookmark